src/seedsigner/views/seed_views.py

```diff
@@ -1650,6 +1678,9 @@ def __init__(self, seed_num: int = None):
         else:
             self.seed = None
         self.address = self.controller.unverified_address["address"]
+        # override derivation path if it's electrum as bip-39 path was set by default
+        if self.seed and self.seed.is_electrum:
+            self.controller.unverified_address["derivation_path"] = 'm/0h'
```

Comment:
The hard-coding here and the limited Electrum support in `embit_util.get_standard_derivation_path()` seems pretty awkward, but I don't have an improvement suggestion yet.